ksql-engine/src/test/java/io/confluent/ksql/planner/plan/JoinNodeTest.java

```diff
@@ -29,19 +29,21 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
```

Comment:
is it safe to mix Mockito with EasyMock?

Reply:
It is... though its not ideal, and wasn't intentional.

Reply:
I took a look - changing the test from EasyMock isn't trivial. Mainly because its not really clear what a `EasyMock.verify` call is verifying. Just removing these calls may be removing some of what is being tested.

So converting is outside the scope of this PR.

But I don't really want to rewrite what I've added as EasyMock....

How about we live with it with a view to refactoring the whole thing across to Mockito at some point.